packages/react/spec/auto/shadcn-defaults/components/Calendar.tsx

```diff
@@ -26,7 +26,7 @@ function Calendar({ className, classNames, showOutsideDays = true, ...props }: C
         head_cell: "text-muted-foreground rounded-md w-9 font-normal text-[0.8rem]",
         row: "flex w-full mt-2",
         cell: "h-9 w-9 text-center  text-sm p-0 relative [&:has([aria-selected].day-range-end)]:rounded-r-md [&:has([aria-selected].day-outside)]:bg-accent/50 [&:has([aria-selected])]:bg-accent first:[&:has([aria-selected])]:rounded-l-md last:[&:has([aria-selected])]:rounded-r-md focus-within:relative focus-within:z-20",
-        day: cn("", "h-9 w-9 p-0 font-normal aria-selected:opacity-100 text-justify"),
```

Comment:
I messed with this originally, so ignore. pretty sure their default datepickers will work. doing some integration with Gadget so will test more there.